docs/internal-model.md

```diff
@@ -19,9 +19,125 @@ These are the meta models and their meaning:
 - **StringModel** represent string values.
 - **AnyModel** represent generic values that cannot otherwise be represented by one of the other models.
 
-<p align="center">
-  <img src="./img/MetaModel.png" />
-</p>
+
+```mermaid
+---
+title: Meta Model
+---
+classDiagram
+    direction BT
+    class MetaModel {
+        <<Abstract class>> 
+        String name
+        Any originalInput
+    }
+
+    class TupleValueModel{
+        <<class>>
+        Number index
+        MetaModel value
+    }
+
+    class TupleModel{
+        <<class>>
+        TupleValueModel[] tupleModels
+    }
+
+    class ObjectModel{
+        <<class>>
+        String, ObjectPropertyModel properties
```

Comment:
Looks like there are a few of the properties that are missing their types. In the old image, it's a dictionary with `string` key and `ObjectPropertyModel` value 🙂 

There are a few places this applies 🙂

Reply:
@jonaslagoni There you go! I've fixed it up. 🙃